components/modules/userdetails/ProfileHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+
+import CustomImage from '@/components/ui/image/CustomImage';
+
+interface ProfileHeaderProps {
+  imageUri: string;
+  name?: string;
+  username: string;
+}
+
+const ProfileHeader: React.FC<ProfileHeaderProps> = ({ imageUri, name = null, username }) => {
+  return (
+    <View className="flex-row w-full justify-start px-3 space-x-3">
+      <CustomImage imageUri={imageUri} size={70} />
```

Comment:
why we use 70 and 150 below and not the sizes that we set small, medium, large etc ?

Reply:
because  in mocks we use 70 and 150 for those components.  Should i change it?

Reply:
done